Justfile

```diff
@@ -120,7 +120,7 @@ build-cohort-extractor:
     set -euo pipefail
 
     [[ -v CI ]] && echo "::group::Build cohort-extractor (click to view)" || echo "Build cohort-extractor"
-    docker build . -t cohort-extractor-v2
```

Comment:
nit: There are a few other `cohort-extractor` references in the Justfile that could be updated while editing this file here.

Equally, could be a separate PR.